client/my-sites/upgrades/domain-search/domain-search.jsx

```diff
@@ -149,13 +142,17 @@ var DomainSearch = React.createClass( {
 			<Main className={ classes }>
 				<SidebarNavigation />
 				{ content }
+				<QueryProductsList />
 			</Main>
 		);
 	}
-} );
+}
 
-module.exports = connect(
+export default connect(
 	( state ) => ( {
-		domainsWithPlansOnly: currentUserHasFlag( state, DOMAINS_WITH_PLANS_ONLY )
+		selectedSite: getSelectedSite( state ),
+		domainsWithPlansOnly: currentUserHasFlag( state, DOMAINS_WITH_PLANS_ONLY ),
+		isSiteUpgradeable: isSiteUpgradeable( state, getSelectedSiteId( state ) ),
+		productsList: state.productsList.items,
```

Comment:
We should definitely introduce a selector for this one (in a separate PR, of course).